src/documents/forms/DocumentTitleValidationIndicators.js

```diff
@@ -31,8 +31,8 @@ const DocumentTitleValidationIndicators = ({errors}) => {
             errors.file_name && errors.file_name.blacklisted ? 'teal' : 'black'
```

Comment:
I think the consensus was to remove these indicators and instead use a field description on the Document Title field as Allison specified